src/dlsproto/client/DlsClient.d

```diff
@@ -744,6 +744,38 @@ public class DlsClient : IClient
         this.node_handshake = new NodeHandshake;
     }
 
+     /***************************************************************************
+
+        Constructor with support for only the neo protocols. Accepts only
```

Comment:
"the neo protocol"

Reply:
Sorry, it should be `for neo and legacy protocols`.